scripts/run_samples.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euo pipefail
+
+# First stab at executing sample projects as E2E tests.
+# This script will:
+# 1. Build all packages
+# 2. Pack all packages into tarballs in a common directory `packs` (to emulate a publish)
+# 3. Visit each sample project and run `npm install` (note it's npm not pnpm because we're planning to test compatibility with older node versions as well)
+# 4. npm run loctool in each of them
+
+echo "Building all packages"
+pnpm build
+
+echo "Packing all packages into common directory"
+rm -r packs
+pnpm -r exec pnpm pack --pack-destination ../../packs/
+
+# TEMP: Rename tarballs to strip version (i.e. everything after the last -) - this will be removed after updating to pnpm 10
+for f in packs/*.tgz; do
+    tar_name="$(basename "$f" .tgz)"
+    package_name="${tar_name%-*}"
+    mv "$f" "packs/$package_name.tgz"
+done
+
+# Run npm install in each sample project
+for sample in packages/ilib-loctool-*/samples/*; do
+    echo "Processing sample project $sample"
+    pushd "$sample"
+    npm install
+    node_modules/.bin/loctool localize .
```

Comment:
Some samples are loctool, some are lint, so we need a script that will run the appropriate thing for the current package. Something like this (doesn't have to be exactly this, but just to illustrate what I mean):

```suggestion
   npm run sample
```
What do you think?